module.json

```diff
@@ -3,9 +3,9 @@
     "name": "Calendar",
     "description": "Calendar for spaces or user profiles.",
     "keywords": ["calendar"],
-    "version": "1.0.13",
+    "version": "1.0.14",
```

Comment:
If we require a new major version, we should increase version at least to `1.1.0`. (So we can still provide patches for older versions)

Reply:
done in commit https://github.com/humhub/humhub-modules-calendar/commit/45b339a206d70f60fab44a91c7013ad5e0dfe204